code/modules/mob/living/carbon/carbon.dm

```diff
@@ -643,7 +643,7 @@
 	. = ..()
 	if(!istype(loc, /turf/space))
 		for(var/obj/item/I in get_all_slots())
-			if(I.flags & SLOWDOWN_WHEN_CARRIED)
+			if(I == src.back)
```

Comment:
you're... not checking the flag at all? any item worn on the back will slow you?

Reply:
My understanding is that if it's on the back, it slows you at half the rate

Reply:
`SLOWDOWN_WHEN_CARRIED` is supposed to be just for items in your hand. It's extremely confusing but I think I got it right this time

Reply:
Ah, I see what you mean. Yes, this is a little unintuitive.